portfolio/src/main/java/com/google/sps/Data/UserAuthenticationData.java

```diff
@@ -20,10 +20,12 @@
   private final boolean loggedIn;
   private final String authenticationUrl;
   private final String email;
+  private final String nickname; // User-provided display name.
 
   public UserAuthenticationData(boolean loggedIn, String authenticationUrl, String email) {
```

Comment:
I just noticed that you may need to pass in nickname here!

Reply:
Yup, just ran into this problem😅 Done!